examples/010_harmonic_example.py

```diff
@@ -0,0 +1,241 @@
+"""
+.. _harmonic_example:
+
+Postprocess a harmonic analysis
+--------------------------------------
+
+.. note::
+
+    This example is work in progress. Please open a github issue if something is unclear.
+
+This example shows how to evaluate failure criteria for a harmonic simulation.
+It shows how to create a phase sweep to compute the maximum IRF in the frequency-phase
+space and shows how to identify the critical failure mode and the critical layer.
+
+"""
+
+
+# %%
+# Set up analysis
+# ~~~~~~~~~~~~~~~
+# Setting up the analysis consists of loading Ansys libraries, connecting to the
+# DPF server, and retrieving the example files.
+#
+# Load Ansys libraries.
+import ansys.dpf.core as dpf
+import matplotlib.pyplot as plt
+
+from ansys.dpf.composites.composite_model import CompositeModel
+from ansys.dpf.composites.constants import FailureOutput
+from ansys.dpf.composites.example_helper import get_continuous_fiber_example_files
+from ansys.dpf.composites.failure_criteria import (
+    CombinedFailureCriterion,
+    FailureModeEnum,
+    MaxStressCriterion,
+    TsaiWuCriterion,
+)
+from ansys.dpf.composites.layup_info.material_operators import get_material_operators
+from ansys.dpf.composites.server_helpers import connect_to_or_start_server
+from ansys.dpf.composites.unit_system import get_unit_system
+
+# Todo: Discuss influence of damping factor
+# Todo: Currently the CompositeDefinition file from the modal analysis needs
```

Comment:
Do you plan to improve the get_composite_files_from_workbench_result_folder later or in a separate PR?
For instance by adding the option has_upstream_analysis.